client/src/components/arrowAfter.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { DivArrowAfter } from '../styles/styles.jsx';
+
+class ArrowAfter extends React.Component {
```

Comment:
This is a stateless component because you don't use state. Also, you can reuse this arrow so that you only have one arrow component. You can have an if/else statement that can make it reusable for both instances.